manifests/etcdquorumguard/disruption-budget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  namespace: kube-system
+  name: etcd-quorum-guard
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: etcd-quorum-guard
```

Comment:
this must be `k8s-app` based on https://github.com/openshift/machine-config-operator/pull/613/files#diff-0c1db25c8965ca5514e8f958eb10f61dR20

Reply:
OK